src/VolumeOfFluidEquationSystem.C

```diff
@@ -270,12 +281,12 @@ VolumeOfFluidEquationSystem::register_inflow_bc(
 
   stk::mesh::MetaData& meta_data = realm_.meta_data();
 
-  // register boundary data; gamma_bc
+  // register boundary data; vof_bc
```

Comment:
Should we just remove these comments (and the following)? The code is pretty explicit I think.

Reply:
I purged a bunch of these comments. We should probably do a pass through the code base to get more of them since this is quite common throughout the code.